### ISSUE TITLE
Update of dependent package "@discordjs/opus"

### ISSUE THREAD

**Issue by kokarare1212** [amishshah/prism-media#92]
#### Issue:
It uses the dependency package "ansi-regex" (https://github.com/advisories/GHSA-93q8-gq69-wqmw),
which is vulnerable due to an outdated version of the dependency package "@discordjs/opus".
The version of the dependent package "@discordjs/opus" needs to be updated.
```cmd
# npm audit report

ansi-regex  >2.1.1 <5.0.1
Severity: moderate
 Inefficient Regular Expression Complexity in chalk/ansi-regex - https://github.com/advisories/GHSA-93q8-gq69-wqmw
fix available via `npm audit fix`
node_modules/ansi-regex
  strip-ansi  4.0.0 - 5.2.0
  Depends on vulnerable versions of ansi-regex
  node_modules/strip-ansi
    string-width  2.1.0 - 4.1.0
    Depends on vulnerable versions of strip-ansi
    node_modules/string-width
```
```cmd
`-- @discordjs/opus@0.5.3
  `-- @discordjs/node-pre-gyp@0.4.2
    `-- npmlog@5.0.1
      `-- gauge@3.0.1
        `-- strip-ansi@4.0.0
          `-- ansi-regex@3.0.0
```

#### Further details:
- Operating System: Windows 11 (22000.318)
- Node.js version: 16.13.0
- Commit I'm using: https://github.com/amishshah/prism-media/tree/v1.3.2

**Comment by fredkilbourn** [comment]
This issue is further upstream and needs to be fixed first: https://github.com/discordjs/node-pre-gyp/issues/5

**Comment by fredkilbourn** [comment]
The current versions of @discordjs/opus are also vulnerable.  Once `@discordjs/node-pre-gyp` is fixed `@discordjs/opus` will also need to reference that fix.  Once that happens `prism-media` can pull in the update.

**Comment by kokarare1212** [comment]
Yes, I understand.
I will wait until a fixed version is released.

**Comment by fredkilbourn** [comment]
I think they fixed it a while ago...

**Comment by darahask** [comment]
@fredkilbourn is it possible to update the discord opus version dependency then.

**Comment by fredkilbourn** [comment]
This issue is referencing an issue from last year, not related to the CVE from this month.

**Comment by amishshah** [comment]
Fixed in v1.3.4
